### ISSUE TITLE
Docker image errors with startup loop - Fails with initializing/started/not responding/error

### ISSUE THREAD

**Issue by mtjoseph** [comassky/roon-docker-server#1]
Any ideas on how to resolve or enable more detailed debug output?

After starting, Docker log file shows the following:
Warn: get lock file path: /tmp/.rnsgem0-
Trace: [childprocess] using unix child process
Initializing
Started
Not responding
Error
Initializing
Started
Not responding

**Comment by comassky** [comment]
Hi,

Thanks for using this image.

It seems to be a global Roon issue, others platforms (even Roon Core) has this issue

https://community.roonlabs.com/t/problem-roon-core-synology/104675/9
https://community.roonlabs.com/t/remote-wont-connect-to-qnap-after-update-to-roon-1-7-build-511/90392

**Comment by mtjoseph** [comment]
I resolved it by doing a clean install of the VM on FreeNas. My best guess is filesystem nd/or docker configuration corruption. I was not able to determine specifically what the issue was. 

For any one else that may find themselves in this whole. I followed these instructions for Freenas 11.3
https://blog.hugopoi.net/2020/03/01/install-rancheros-on-freenas-11-3/

Once the VM is up and running, I installed Portainer and  comassky / roon-docker-server 

Portainer: 
docker run -d -p 9000:9000  --name portainer --restart always --network=host -v /var/run/docker.sock:/var/run/docker.sock -v portainer_data:/data portainer/portainer-ce

roon-docker-server: 
docker run -d --net="host" --name roonserver --restart always -e TZ="America/Los_Angeles" -v roon-app:/app -v roon-data:/data -v roon-music:/music -v roon-backups:/backup comassky/roon:latest

Now off to do some Easy Listening with Roon!
